docs/security.rst

```diff
@@ -0,0 +1,50 @@
+.. highlight:: javascript
+
+.. _security-reference:
+
+********
+Security
+********
+
+Linz comes with a number of security features out of the box including:
+
+- CSRF Protection
+
+CSRF Protection
+----------------
+
+CSRF protection helps prevent unauthorized commands that are transmitted from a user that the web application trusts.
+
+You can read more about CSRF here https://en.wikipedia.org/wiki/Cross-site_request_forgery.
+
+Customising CSRF
+================
+
+Linz uses the csurf module to provide CSRF protection. https://github.com/expressjs/csurf
```

Comment:
Could you please link the text _csurf module_ to the link you provided using the RST external link syntax... http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets